src/amo/components/AddonReviewListItem/styles.scss

```diff
@@ -36,9 +36,11 @@
   a:visited,
   a:hover,
   a:active {
-    color: $blue-50;
-    font-weight: normal;
-    text-decoration: none;
+    &:not(.Button) {
```

Comment:
I don't see `button` in the selector list. Why was this needed?

Reply:
The buttons are `a` tags here (and in the other place) because they have a `href` attribute.

Reply:
Ah, the login button is an anchor. I didn't think of that.